### PR TITLE
proposed change for street names

### PR DESCRIPTION
**Pull request by kpwebb**
@migurski here's my first attempt at including street names into the metadata tiles

### CODE REVIEW

File: proto/sharedstreets.proto [hunk 1]
```diff
@@ -37,10 +37,13 @@ message WaySection {
   bool roundabout = 4;
   bool link = 5;
   repeated uint64 nodeIds = 6;
+  
+  string name  = 7; // name only stored here if different for each way section, otherwise captured in OSMMetadata
```
Comment (DenisCarriere) [inline proto/sharedstreets.proto:41]: 👍 Seems reasonable to me

**Comment by migurski** [comment]
Looks good to me!

**Comment by DenisCarriere** [comment]
Once merged, I'll update the following repos:

- [ ] [sharedstreets-pbf](https://github.com/sharedstreets/sharedstreets-pbf)
- [ ] [sharedstreets-types](https://github.com/sharedstreets/sharedstreets-types)

**Comment by kpwebb** [comment]
Great! Merging now and regenerating the tiles

**Comment by DenisCarriere** [comment]
@kpwebb 👍 Great stuff!

**Pull request merged**